cis194/week1/robert-chang/Homework01/src/Homework01.hs

```diff
@@ -0,0 +1,21 @@
+module Homework01
+    (
+      toDigits,
+      toDigitsRev,
+      doubleEveryOther, 
+      sumDigits
+    ) where
```

Comment:
In Haskell, the convention is to align commas with brackets:
```Haskell
    ( toDigits
    , toDigitsRev
    , doubleEveryOther
    , sumDigits )
```

Reply:
thanks! changed.